dozer/cogs/fun.py

```diff
@@ -111,7 +116,7 @@ async def battle(self, ctx: DozerContext, opponent: discord.Member, delete_resul
     @guild_only()
     @discord.ext.commands.cooldown(1, 5, BucketType.channel)
     @discord.ext.commands.max_concurrency(1, per=BucketType.channel, wait=False)
-    @command()
+    @commands.command()
```

Comment:
No

Reply:
Pretty sure the @commands.command part comes from how discord.ext commands are being imported